lessons/pydata/api/index.ipynb

```diff
@@ -0,0 +1,2168 @@
+{
```

Comment:
<p>Co takhle tady raději použít <code>pprint</code>  než převod zpět na řetězec?</p><p>Pokud by to takto mělo zůstat, zvážil bych lepší (popisnější) jméno pro proměnnou a zrušil mezery okolo <code>=</code>  u pojmenovaných argumentů.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>

Reply:
<p>mezery pryč</p>